src/App.tsx

```diff
@@ -1,7 +1,35 @@
 import React from 'react';
 
-export const App: React.FC = () => (
-  <div className="App">
-    <p className="App__message">The last pressed key is [Enter]</p>
-  </div>
-);
+type State = {
+  message: string | null;
+};
+
+export class App extends React.Component<{}, State> {
+  state: Readonly<State> = {
+    message: 'Nothing was pressed yet',
+  };
+
+  componentDidMount() {
+    document.addEventListener('keyup', this.pressedKey);
+  }
+
+  componentWillUnmount() {
+    document.removeEventListener('keyup', this.pressedKey);
+  }
+
+  pressedKey = (event: KeyboardEvent) => {
```

Comment:
methods names must start from verbs

take setMessage or handleMessage

it was in previouses checklists

Reply:
done